lib/src/builder.dart

```diff
@@ -123,6 +127,10 @@ class _Builder extends Builder {
 
     var genPartContent = contentBuffer.toString();
 
+    if (_header.isNotEmpty) {
+      genPartContent = '${_header}\n$genPartContent';
```

Comment:
nit: move this up above and add it to `contentBuffer` right after creation, should avoid some extra string copying and be a bit cleaner.

Reply:
kk